flox/flock/flock.py

```diff
@@ -233,10 +234,26 @@ def children(self, node: FlockNode | FlockNodeID | int) -> Generator[FlockNode]:
             yield FlockNode(
                 idx=child_idx,
                 kind=self.topo.nodes[child_idx]["kind"],
+                path=self.topo.nodes[child_idx]["path"],
                 globus_compute_endpoint=self.topo.nodes[child_idx][gce],
                 proxystore_endpoint=self.topo.nodes[child_idx][pse],
             )
 
+    @staticmethod
+    def node_path(topo: nx.DiGraph, node: FlockNode | FlockNodeID | int) -> str:
+        if isinstance(node, FlockNode):
+            idx = node.idx
+        else:
+            idx = node
+        path = ""
```

Comment:
`pathlib` is very filesystem oriented, even when using supposedly computation-only [`PurePath`s](https://docs.python.org/3/library/pathlib.html#pure-paths), so I'm wary to use them here in case we get different string outputs on different platforms